src/main/java/ProjectMgt/ProjectMgtInputBoundary.java

```diff
@@ -0,0 +1,5 @@
+package ProjectMgt;
+
+public interface ProjectMgtInputBoundary {
+    ProjectMgtRequestModel createTask(ProjectMgtRequestModel requestModel);
```

Comment:
Did you mean createProject, since project management needs to initialize a project instead of task

Reply:
as in createTask returns ResponseModel